src/features/blockchain/config/prompts/defaultPrompts.ts

```diff
@@ -1,5 +1,5 @@
 export const defaultIntroText = `
-What can I help you with?`;
+How can I help you with Web3?`;
```

Comment:
Did this come from product? I feel like this is a narrative kind of thing. Fine with the changes for now.